portal/static/js/flask_user/ConfirmIdentity.js

```diff
@@ -0,0 +1,27 @@
+$(document).ready(function() {
+    $("#confirmIdentityModal").modal({
+        "show":true,
+        "backdrop": "static",
+        "focus": true,
+        "keyboard": false
+    });
+    $("#btnConfirmIdentity").on("click", function(e) {
+        e.stopPropagation();
+        /*
+         * record an audit event before redirecting user to survey
+         */
+        $.ajax({
+            type: "POST",
+            url: "/api/auditlog",
+            data: {
+                "message": "User confirmed identity as survey taker"
```

Comment:
I'll note, the user & subject of the audit will be pulled from the current user.